packages/syft/src/syft/service/project/project_service.py

```diff
@@ -49,19 +49,6 @@ def create_project(
     ) -> Union[SyftSuccess, SyftError]:
         """Start a Project"""
         try:
-            # Check if the project with given id already exists
-            project_id_check = self.stash.get_by_uid(
-                credentials=context.node.verify_key, uid=project_id
-            )
-
-            if project_id_check.is_err():
-                return SyftError(message=f"{project_id_check.err()}")
-
-            if project_id_check.ok() is not None:
-                return SyftError(
-                    message=f"Project with id {project_id} already exists. Kindly use a different id"
-                )
-
```

Comment:
`UID` is anyway unique at stash level, so this will already be handled. Let me know if there's a case that I may be unaware of.